main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"flag"
+	"log"
+
+	"github.com/canonical/paascharmgen/internal/paascharm"
+)
+
+const (
+	defaultCharmcraftLocation = "./"
+	defaultPackageName        = "charmconfig"
+	defaultOutputFile         = "charmconfig.go"
```

Comment:
Perhaps instead of charm this would be app config or something like that? Since the user would use it in their app?

Reply:
updated to appconfig for both of them